### PR TITLE
Script for defect 1902

### PR DESCRIPTION
**Pull request by HSavynetska**
ATF Test Scripts to check [#1902](https://github.com/smartdevicelink/sdl_core/issues/1902)

This PR is **ready** for review.

### Summary
A test script to verify that issue is fixed

### ATF version
[develop](https://github.com/smartdevicelink/sdl_atf/tree/develop)

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Comment by dboltovskyi** [comment]
Livio team please do not review this PR for now since it hasn't passed our internal review yet.  
Review and appropriate updates going be done till 1st of November.
Thank you.

File: test_scripts/Defects/5_0/1902_Media_app_and_navi_app_both_must_get_AUDIBLE_streaming_state_after_successfull_resumption.lua [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+-- Media app and navi app both must get AUDIBLE streaming state after successfull resumption
+-- Precondition:
+-- SDL and HMI are started.
+-- In case:
+-- 1) Media app registers and gets FULL/LIMITED and AUDIBLE
+-- 2) Navigation app registers and satisfies all conditions for HMILevel resumption
+-- Expected result:
+-- 1) SDL must resume navi app to appropriate (according to existing req-s) and AUDIBLE audioStreamingState (by sending OnHMIStatus notification to mobile app).
+-- Actual result:
+-- SDL does not resume HMILevel and audioStreamingState of navigation app.
```
Comment (dboltovskyi) [inline test_scripts/Defects/5_0/1902_Media_app_and_navi_app_both_must_get_AUDIBLE_streaming_state_after_successfull_resumption.lua]: @HSavynetska Please remove actual result

File: test_scripts/Defects/5_0/1902_Media_app_and_navi_app_both_must_get_AUDIBLE_streaming_state_after_successfull_resumption.lua [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+runner.Step("Activate media app", common.activateApp)
+
+runner.Title("Test")
+runner.Step("IGNITION_OFF", ignitionOff)
+runner.Step("Close mobile connection", cleanSessions)
+runner.Step("Ignition On", common.start)
+runner.Step("Register media app", common.registerAppWOPTU)
+runner.Step("checkResuming media app", checkResumingMediaApp)
+runner.Step("Register navi app", common.registerAppWOPTU, {2})
+runner.Step("checkResuming navi app", checkResumingNaviApp)
```
Comment (dboltovskyi) [inline test_scripts/Defects/5_0/1902_Media_app_and_navi_app_both_must_get_AUDIBLE_streaming_state_after_successfull_resumption.lua]: @HSavynetska Please combine registration and resumption steps

**Comment by dboltovskyi** [comment]
Livio team please notice script is ready for review.

**Comment by HSavynetska** [comment]
This test script is not relevant.
The defect is not reproduced on the current develop

Environment:
SDL build: https://github.com/smartdevicelink/sdl_core/tree/develop ( 891497b )
ATF: develop branch

**Pull request closed without merging**